code/src/Components/Form.js

```diff
@@ -0,0 +1,123 @@
+import React, { useState } from "react";
+
+
+
+const Form = () => {
+  const [genre, setGenre] = useState('');
+  const [time, setTime] = useState('');
+  const [experience, setExperience] = useState('');
+  const [surveyComplete, setSurveyComplete] = useState(false);
+
+  const timeChoice = ['Full time', 'Part time', 'Temporary project']
+  const experienceChoice = ['Under education', 'Junior', 'Senior']
+
+  const isSurveyComplete = () => {
+    if(time !== '' && genre !== '' && experience !== '' ){
+        return true;
+     }
+    return false; 
+  }
+
+  const handleSubmit = (event) => {
+    event.preventDefault() 
+    
+    if (isSurveyComplete()) {
+      setSurveyComplete(true)
+    } else {
+      alert("Please choose options for each question")
+    }
+  }
+
+const onGenreChange = (e) => {
+  setGenre(e.target.value);
+}
+
+const onTimeChange = (e) => {
+  setTime(e.target.value);
+}
+
+const onExperienceChange = (e) => {
+  setExperience(e.target.value);
+}
+
+if (surveyComplete) {
+  return (
+    <div className="form">
+      <p>Thank you for filling out this survey.</p>
+      <div className="circle-1"></div>
+      <p>We have noticed that you are looking for a {time} position, within the {genre} genre which accepts applications from people with {experience} experience. We will process your request.</p> 
+    </div>
+    
+  )
+ 
+} else {
+
+  return (    
+  
+    <form className="form" onSubmit={handleSubmit}>
+      <p tabIndex="0"><b>What kind of job are you looking for?</b></p>
+      <div className="time-container">
+      {timeChoice.map((time) => (
+           <label key={time} htmlFor={time}>
+           <input 
+             id={time}
+             name="time" 
+             type="radio" 
+             value={time} 
+             required
+             onChange={onTimeChange}
+             >
+           </input>
+           {time} 
+           </label>
+          ))}
+      </div>
```

Comment:
Please have a look at the indentation here, it's a little bit off. I would suggest you to move in {timeChoice.....} and maybe also {time}.  The input tag I would also suggest to move in. On line 69 close tag should go one step to the left <-- to match the input opening tag.  On line 73 I would put the (({ to match the opening of the {timeChoice...}. Am I being confusing? hope not 🙃